code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -1,16 +1,20 @@
 /obj/item/gun/energy/pulse
 	name = "pulse rifle"
-	desc = "A heavy-duty, multifaceted energy rifle with three modes. Preferred by front-line combat personnel."
+	desc = "A heavy-duty, expensive, multifaceted energy rifle with three modes. The standard for heavy specialist Nanotrasen security."
```

Comment:
Wording and flow.

"A top-of-the-line, heavy-duty, multifaceted energy rifle with three firing modes. The gold standard for Nanotrasen's heavier security specialists."